examples/cpp/pyperf/PyPerfNativeStackTrace.cc

```diff
@@ -94,20 +94,26 @@ NativeStackTrace::NativeStackTrace(uint32_t pid, const unsigned char *raw_stack,
     unw_word_t offset;
     unw_word_t ip;
     struct bcc_symbol resolved_symbol;
+    resolved_symbol.module = nullptr;
     unw_get_reg(&cursor, UNW_REG_IP, &ip);
     if (procSymbols && procSymbols->resolve_addr(ip, &resolved_symbol, true)) {
         snprintf(buf, buf_size, "%s (%s)", resolved_symbol.demangle_name, resolved_symbol.module);
+        // let bcc take care of freeing memory allocated by __cxa_demangle()
+        bcc_symbol_free_demangle_name(&resolved_symbol);
```

Comment:
Good call haha. 

I checked `get_stack_symbol` (which uses this API as well) and it seems to be the only "cleanup" function we should be using